eth/tools/builder/chain/builders.py

```diff
@@ -234,6 +237,8 @@ def dao_fork_at(dao_fork_block_number: BlockNumber,
 petersburg_at = fork_at(PetersburgVM)
 istanbul_at = fork_at(IstanbulVM)
 
+latest_mainnet_at = petersburg_at
```

Comment:
:+1: to not making this magic so tests don't suddenly start failing without this being updated explicitly.